packages/pancake-profile-sdk/src/utils/getRpcUrl.ts

```diff
@@ -0,0 +1,19 @@
+// Array of available nodes to connect to
+export const nodes = [
```

Comment:
Just a note that any improvement we make to this node-assignment in the FE ought to be translated here, or maybe even ideally hosted in its own package..?

Reply:
Might be worth putting nodes array into the constructor when using SDK, you're right, this is not very future-proof. I'll add an optional argument for constructor to supply own nodes.

And of course this is not be-all and end-all of Profile SDK, if we decide to somehow completely redesign how nodes are used that could be changed in future updates.